release_notes/ocp_3_9_release_notes.adoc

```diff
@@ -0,0 +1,434 @@
+
+If the user is running as root on the localhost, or connection to the root user on the remote hosts instead of using become, no change should be noticed.
+
+[discrete]
+[[unqualified-image-specs]]
+===
+unqualified image specs now default to docker.io and require apiserver configuration to resolve to different registries.
+
+WIP in https://bugzilla.redhat.com/show_bug.cgi?id=1518378 to determine what this config looks like and how we will migrate existing unqualified pods
+
+[discrete]
+[[SchedueldJob-objects-not-supported]]
+=== batch/v2alpha1 SchedueldJob objects are no longer supported
+
+batch/v2alpha1 SchedueldJob objects are no longer supported. Use CronJobs instead.
+
+[discrete]
+[[autoscaling-API-group-removed]]
+===  The autoscaling/v2alpha1 API group has been removed
+The autoscaling/v2alpha1 API group has been removed
+
+[discrete]
+[[start-node-requires-swap-to-be-disabled]]
+The openshift start node requires swap to be disabled. This is already done as part of the ansible node install.
+
+
+[[ocp-39-bug-fixes]]
+== Bug Fixes
+
+This release fixes bugs for the following components:
+
+*Authentication*
+
+*
+
+*Builds*
+
+
+*Command Line Interface*
+
+
+*Containers*
+
+
+*Deployments*
+
+
+*Images*
+
+*Image Registry*
+
+
+*Installer*
+
+*Logging*
+
+
+*Web Console*
+
+
+*Master*
+
+
+*Metrics*
+
+
+*Networking*
+
+*Pod*
+
+
+*Routing*
+
+
+*Service Broker*
+
+
+*Storage*
+
+
+*Templates*
+
+
+*Upgrade*
+
+
+[[ocp-37-technology-preview]]
+== Technology Preview Features
+
+Some features in this release are currently in Technology Preview. These
+experimental features are not intended for production use. Please note the
+following scope of support on the Red Hat Customer Portal for these features:
+
+https://access.redhat.com/support/offerings/techpreview[Technology Preview
+Features Support Scope]
+
+The following new features are now available in Technology Preview:
+
+- xref:ocp-39-device-manager[Device Manager]
+- GPU Support
+- Persistent Volume Resize
+
+The following features that were formerly in Technology Preview from a previous
+{product-title} release are now fully supported:
+
+- xref:../dev_guide/cron_jobs.adoc#dev-guide-cron-jobs[Cron Jobs]
+- xref:ocp-39-crio[CRI-O]
+- xref:ocp-39-cli-plug-ins[CLI Plug-ins]
+- xref:../dev_guide/deployments/kubernetes_deployments.adoc#dev-guide-kubernetes-deployments-support[Kubernetes
+Deployments Support]
+- `StatefulSets`
+- xref:../admin_guide/quota.adoc#limited-resources-quota[Require Explicit Quota to Consume a Resource]
+- xref:../architecture/additional_concepts/storage.adoc#pv-mount-options[Mount Options]
+- xref:../install_config/install/advanced_install.adoc#advanced-install-configuring-system-containers[Installation of etcd, Docker Daemon, and Ansible Installer as System Containers]
+- xref:../install_config/install/advanced_install.adoc#running-the-advanced-installation-system-container[Running OpenShift Installer as a System Container]
```

Comment:
We don't have docs (from what I can tell) on how to migrate or upgrade a cluster so that it uses system containers. 
Will we have such a guide? If not should we call out that the use of system containers are only supported for new installs (as this is now just leaving TP)?

Reply:
In my mind "upgrading" to system containers likely would be to bring up new nodes using system containers and sun setting old style nodes. Adding in @giuseppe for his POV.